tests/integration/test_logtest/test_ruleset_refresh/test_alert_labels.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import shutil
+
+import pytest
+import yaml
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools.configuration import set_section_wazuh_conf
+import wazuh_testing.tools.configuration as conf
+
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+# Configurations
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+messages_path = os.path.join(test_data_path, 'log_alert_level.yaml')
+with open(messages_path) as f:
+    test_cases = yaml.safe_load(f)
+
+# Variables
+logtest_path = os.path.join(os.path.join(WAZUH_PATH, 'queue', 'sockets', 'logtest'))
+receiver_sockets_params = [(logtest_path, 'AF_UNIX', 'TCP')]
+
+
+# Fixtures
+@pytest.fixture(scope='function')
+def configure_rules_list(get_configuration, request):
+    """
+    Configure a custom rules and log alert level for testing.
+    Restart Wazuh is not needed for applying the configuration is optional.
+    """
+
+    # configuration for testing
+    rules_dir = os.path.join('/var/ossec/',  get_configuration['rule_dir'])
```

Comment:
```suggestion
    rules_dir = os.path.join(WAZUH_PATH,  get_configuration['rule_dir'])
```

We have `WAZUH_PATH` imported, we could use it here.